frontend/src/Pages/StudentManagement.tsx

```diff
@@ -302,72 +314,51 @@ export default function StudentManagement() {
-                type={ModalType.Edit}
-                item="Resident"
-                form={
-                    targetUser ? (
-                        <EditUserForm
-                            onSuccess={hanldleEditUser}
-                            user={targetUser}
-                        />
-                    ) : (
-                        <div>No user defined!</div>
-                    )
-                }
+                target={targetUser?.target}
             />
-            <Modal
+            <TextOnlyModal
                 ref={deleteUserModal}
-                type={ModalType.Confirm}
-                item="Delete Resident"
-                form={
-                    <DeleteForm
-                        item="User"
-                        onCancel={handleDeleteUserCancel}
-                        onSuccess={() => void deleteUser()}
-                    />
+                type={TextModalType.Delete}
+                title={'Delete Resident'}
+                text={
+                    'Are you sure you would like to delete this resident? This action cannot be undone.'
                 }
+                onSubmit={() => void deleteUser()}
+                onClose={() => void handleCancelModal(deleteUserModal)}
             />
-            <Modal
+            <TextOnlyModal
                 ref={resetUserPasswordModal}
-                type={ModalType.Confirm}
-                item="Reset Password"
-                form={
-                    <ResetPasswordForm
-                        user={targetUser}
-                        onCancel={handleResetPasswordCancel}
-                        onSuccess={handleDisplayTempPassword}
-                    />
-                }
+                type={TextModalType.Confirm}
+                title={'Reset Password'}
+                text={`Are you sure you would like to reset ${targetUser?.target.name_first + ' ' + targetUser?.target.name_last}'s password?`}
+                onSubmit={() => void getTempPassword()}
+                onClose={() => void handleCancelModal(resetUserPasswordModal)}
             />
-            <Modal
+            <TextOnlyModal
                 ref={showUserPassword}
-                type={ModalType.Show}
-                item={'New Password'}
-                form={
-                    <ShowTempPasswordForm
-                        tempPassword={tempPassword}
-                        userName={
-                            targetUser
-                                ? `${targetUser.name_first} ${targetUser.name_last}`
-                                : undefined
-                        }
-                        onClose={handleShowPasswordClose}
-                    />
-                }
-            />
+                type={TextModalType.Information}
+                title={'New Password'}
+                text={`Copy this password now. If you lose it, you'll need to
+                        regenerate it to get a new one.`}
```

Comment:
Can you update this text to

"Copy your password now. If you lose it, you'll need to generate a new one."